File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Invs/INV2003U00ExportCSVInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Invs
{
    public class INV2003U00ExportCSVInfo
    {
        private String _rowNum;
        private String _churiDate;
        private String _exportTime;
        private String _jaeryoCode;
        private String _jaeryoName;
        private String _chulgoQty;
        private String _ipgoDanuiName;
        private String _chulgoType;
        private String _updId;
        private String _exportCode;
        private String _comment;

        public String RowNum
        {
            get { return this._rowNum; }
            set { this._rowNum = value; }
        }

        public String ChuriDate
        {
            get { return this._churiDate; }
            set { this._churiDate = value; }
        }

        public String ExportTime
        {
            get { return this._exportTime; }
            set { this._exportTime = value; }
        }

        public String JaeryoCode
        {
            get { return this._jaeryoCode; }
            set { this._jaeryoCode = value; }
        }

        public String JaeryoName
        {
            get { return this._jaeryoName; }
            set { this._jaeryoName = value; }
        }

        public String ChulgoQty
        {
            get { return this._chulgoQty; }
            set { this._chulgoQty = value; }
        }

        public String IpgoDanuiName
        {
            get { return this._ipgoDanuiName; }
            set { this._ipgoDanuiName = value; }
        }

        public String ChulgoType
        {
            get { return this._chulgoType; }
            set { this._chulgoType = value; }
        }

        public String UpdId
        {
            get { return this._updId; }
            set { this._updId = value; }
        }

        public String ExportCode
        {
            get { return this._exportCode; }
            set { this._exportCode = value; }
        }

        public String Comment
        {
            get { return this._comment; }
            set { this._comment = value; }
        }

        public INV2003U00ExportCSVInfo() { }

        public INV2003U00ExportCSVInfo(String rowNum, String churiDate, String exportTime, String jaeryoCode, String jaeryoName, String chulgoQty, String ipgoDanuiName, String chulgoType, String updId, String exportCode, String comment)
        {
            this._rowNum = rowNum;
            this._churiDate = churiDate;
            this._exportTime = exportTime;
            this._jaeryoCode = jaeryoCode;
            this._jaeryoName = jaeryoName;
            this._chulgoQty = chulgoQty;
            this._ipgoDanuiName = ipgoDanuiName;
            this._chulgoType = chulgoType;
            this._updId = updId;
            this._exportCode = exportCode;
            this._comment = comment;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/invs/INV2003U00ExportCSVInfo.java
package nta.med.data.model.ihis.invs;

import java.util.Date;

public class INV2003U00ExportCSVInfo {
	private String rowNum;
	private Date churiDate;
	private String exportTime;
	private String jaeryoCode;
	private String jaeryoName;
	private Double chulgoQty;
	private String ipgoDanuiName;
	private String chulgoType;
	private String updId;
	private String exportCode;
	private String comment;

	public INV2003U00ExportCSVInfo(String rowNum, Date churiDate, String exportTime, String jaeryoCode,
			String jaeryoName, Double chulgoQty, String ipgoDanuiName, String chulgoType, String updId,
			String exportCode, String comment) {
		super();
		this.rowNum = rowNum;
		this.churiDate = churiDate;
		this.exportTime = exportTime;
		this.jaeryoCode = jaeryoCode;
		this.jaeryoName = jaeryoName;
		this.chulgoQty = chulgoQty;
		this.ipgoDanuiName = ipgoDanuiName;
		this.chulgoType = chulgoType;
		this.updId = updId;
		this.exportCode = exportCode;
		this.comment = comment;
	}

	public String getRowNum() {
		return rowNum;
	}

	public void setRowNum(String rowNum) {
		this.rowNum = rowNum;
	}

	public Date getChuriDate() {
		return churiDate;
	}

	public void setChuriDate(Date churiDate) {
		this.churiDate = churiDate;
	}

	public String getExportTime() {
		return exportTime;
	}

	public void setExportTime(String exportTime) {
		this.exportTime = exportTime;
	}

	public String getJaeryoCode() {
		return jaeryoCode;
	}

	public void setJaeryoCode(String jaeryoCode) {
		this.jaeryoCode = jaeryoCode;
	}

	public String getJaeryoName() {
		return jaeryoName;
	}

	public void setJaeryoName(String jaeryoName) {
		this.jaeryoName = jaeryoName;
	}

	public Double getChulgoQty() {
		return chulgoQty;
	}

	public void setChulgoQty(Double chulgoQty) {
		this.chulgoQty = chulgoQty;
	}

	public String getIpgoDanuiName() {
		return ipgoDanuiName;
	}

	public void setIpgoDanuiName(String ipgoDanuiName) {
		this.ipgoDanuiName = ipgoDanuiName;
	}

	public String getChulgoType() {
		return chulgoType;
	}

	public void setChulgoType(String chulgoType) {
		this.chulgoType = chulgoType;
	}

	public String getUpdId() {
		return updId;
	}

	public void setUpdId(String updId) {
		this.updId = updId;
	}

	public String getExportCode() {
		return exportCode;
	}

	public void setExportCode(String exportCode) {
		this.exportCode = exportCode;
	}

	public String getComment() {
		return comment;
	}

	public void setComment(String comment) {
		this.comment = comment;
	}

}
